src/components/Accordion.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import { useState } from "react";
+import { ChevronDownIcon, ChevronLeftIcon } from "@heroicons/react/24/outline";
+
+interface AccordionItem {
+  id: string;
+  label: string;
+  content: string;
+}
+
+interface AccordionProps {
+  items: AccordionItem[];
+}
+
+//This TS use is correct?
```

Comment:
It is correct, but the name Accordion makes me think it will show only one item, but it actually receives a list of items that will be handled internally.

We should consider a name more descriptive, like TaskList, TaskFeed, or something like this